apps/web/src/pages/get-started/tabs/Setup.tsx

```diff
@@ -0,0 +1,367 @@
+import { useEnvironment } from '../../../hooks/useEnvironment';
+import { CodeEditor } from '../CodeBlock';
+import { TextElement } from '../TextElement';
+import { CardButton, CardButtonGroupWrapper } from './CardsButtonGroup';
+import { HStack } from '@novu/novui/jsx';
+import { css } from '@novu/novui/css';
+import { IconOutlineMenuBook } from '@novu/design-system';
+import { useSegment } from '../../../components/providers/SegmentProvider';
+
+export const buildGuides = [
+  {
+    id: 'nextjs',
+    title: 'Next.js',
+    logo: NextJSLogo,
+    steps: [
+      {
+        title: 'Initialize a new project',
+        content: () => {
+          // eslint-disable-next-line react-hooks/rules-of-hooks
+          const { environment } = useEnvironment();
+
+          return <CodeSnippet command={`npx novu@latest init --secret-key=${(environment as any)?.apiKeys[0].key}`} />;
+        },
+      },
+      {
+        title: 'Run the application server',
+        content: () => {
+          return <CodeSnippet command={`cd my-novu-app && npm run dev`} />;
+        },
+      },
+      {
+        title: 'Create a workflow',
+        content: () => {
+          return (
+            <div>
+              <TextElement>
+                To define a new workflow you will use the <Code>@novu/framework</Code> Typescript SDK.
+                <br />
+                In a new file let's define a workflow with a simple email step.
+                <br /> <br />
+              </TextElement>
+
+              <CodeEditor
+                height="200px"
+                readonly
+                setCode={() => {}}
+                code={`import { workflow } from '@novu/framework';
+
+export const myWorkflow = workflow('my-workflow', async ({ step }) => {
+  await step.email('step', async (controls) => {
+    return {
+      subject: controls.subject,
+      body: '<h1>Hello World</h1>',
+    }
+  })
+});
+                `}
+              />
+
+              <HStack gap="50" className={css({ color: 'typography.text.secondary', mt: '12px' })}>
+                <IconOutlineMenuBook />
+                <a href="https://docs.novu.co/workflow/introduction" target={'_blank'}>
+                  Learn more on building workflows
+                </a>
+              </HStack>
+            </div>
+          );
+        },
+      },
+      {
+        title: 'Expose your workflow',
+        content: () => {
+          return (
+            <>
+              <TextElement>
+                Once a workflow has been created, we would need to expose it to the <code>serve</code> function so that
+                it will be visible on the Novu Studio.
+              </TextElement>
+              <br /> <br />
+              <CodeEditor
+                height="120px"
+                readonly
+                setCode={() => {}}
+                code={`import { serve } from '@novu/framework/next';
+import { myWorkflow } from '../../novu/workflows';
```

Comment:
i think we can remove the complexity of telling the user he should import the workflow, all of the code can live in one file for simplicity